packages/bpk-component-calendar/src/BpkCalendarContainer-test.js

```diff
@@ -68,13 +68,15 @@ describe('BpkCalendarContainer', () => {
       selectedDate={new Date(2010, 1, 15)}
     />);
 
-    const grid = calendar.find('BpkCalendarGridTransition');
+    let grid = calendar.find('BpkCalendarGridTransition');
     const nav = calendar.find('BpkCalendarNav');
     const eventStub = { persist: jest.fn() };
 
     expect(grid.prop('month')).toEqual(new Date(2010, 1, 1));
 
     nav.prop('onMonthChange')(eventStub, { month: new Date(2010, 2, 1) });
+    calendar.update();
```

Comment:
See [this item](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#for-mount-updates-are-sometimes-required-when-they-werent-before) to get an idea of why we now have to call .update()` on the root wrapper.

Reply:
I'm getting flashbacks of using Angular and having to put `$scope.$digest()` everywhere! But yeah - I see why this is necessary and the explanation you linked to makes sense.